frontend/src/i18n/en.js

```diff
@@ -152,6 +152,7 @@ const en = {
       create: {
         success: 'Contact successfully saved',
```

Comment:
Can you update to "Contact added successfully"?
To match the image in the issue description https://github.com/CrowdDotDev/crowd.dev/issues/1584